pkgs/applications/networking/ssb/patchwork/default.nix

```diff
@@ -29,16 +30,20 @@ let
     categories = "Network;";
   };
 
-in
-  symlinkJoin {
-    inherit name;
-    paths = [ binary ];
+in stdenvNoCC.mkDerivation {
```

Comment:
I don't really get why we have to use the `NoCC` version here? It has little benefit.